pkgs/applications/editors/idea/default.nix

```diff
@@ -340,12 +340,12 @@ in
 
   datagrip = buildDataGrip rec {
     name = "datagrip-${version}";
-    version = "2016.3.2";
+    version = "2016.3";
```

Comment:
@globin how do we fix the potential downgrade, do I need to perform a full recommit and make a new pull request?